### PR TITLE
gh-121453: Update the `Doc/templates/download.html` download files size estimates and support download `.texi` format.

### PR DESCRIPTION
**Pull request by Wulian233**
Python 3.8 added support for `texinfo` format https://github.com/python/cpython/pull/14606 , but not added link in https://github.com/python/cpython/blob/main/Doc/tools/templates/download.html

And need update the doc download files size estimates. The latest update was in Python 2.7
> According https://www.python.org/ftp/python/doc/3.12.4/

* Issue: gh-121453

----
📚 Documentation preview 📚: https://cpython-previews--121454.org.readthedocs.build/en/121454/download.html

### CODE REVIEW

**Comment by Wulian233** [comment]
@hugovk do you have some time to give this a review? Thanks!

File: Misc/NEWS.d/next/Documentation/2024-07-07-13-02-05.gh-issue-121453.0SJH3S.rst
Comment (hugovk) [inline Misc/NEWS.d/next/Documentation/2024-07-07-13-02-05.gh-issue-121453.0SJH3S.rst:1]: Thanks! PR look good, but let's skip this changelog entry.

File: Doc/tools/templates/download.html [hunk 1]
```diff
@@ -21,23 +21,27 @@ <h1>Download Python {{ release }} Documentation</h1>
...
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-a4.zip">Download</a> (ca. 17 MiB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-pdf-a4.tar.bz2">Download</a> (ca. 17 MiB)</td>
   </tr>
   <tr><td>HTML</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-html.zip">Download</a> (ca. 9 MiB)</td>
-    <td><a href="{{ dlbase }}/python-{{ release }}-docs-html.tar.bz2">Download</a> (ca. 6 MiB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-html.zip">Download</a> (ca. 13 MiB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-html.tar.bz2">Download</a> (ca. 8 MiB)</td>
+  </tr>
+  <tr><td>Texinfo</td>
```
Comment (hugovk) [inline Doc/tools/templates/download.html]: Oh, and let's list Texinfo after Plain Text.

File: Doc/tools/templates/download.html [hunk 2]
```diff
@@ -21,23 +21,27 @@ <h1>Download Python {{ release }} Documentation</h1>
...
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-texinfo.zip">Download</a> (ca. 9 MiB)</td>
+    <td><a href="{{ dlbase }}/python-{{ release }}-docs-texinfo.tar.bz2">Download</a> (ca. 7 MiB)</td>
   </tr>
   <tr><td>Plain Text</td>
```
Comment (hugovk) [inline Doc/tools/templates/download.html]:
```suggestion
  <tr><td>Plain text</td>
```

File: Misc/NEWS.d/next/Documentation/2024-07-07-13-02-05.gh-issue-121453.0SJH3S.rst
Comment (Wulian233) [reply Misc/NEWS.d/next/Documentation/2024-07-07-13-02-05.gh-issue-121453.0SJH3S.rst:1]: Ok

**Comment by hugovk** [comment]
Thanks! I don't think this is something we need to update very regularly (it does say "circa"), but it's okay to do it once in a while after big changes.

**Pull request merged**